Dockerfile.aarch64-android

```diff
@@ -0,0 +1,111 @@
+FROM runmymind/docker-android-sdk:ubuntu-standalone-20230711
+# the above docker file can be found here: https://github.com/mindrunner/docker-android-sdk/blob/main/ubuntu/standalone/Dockerfile
+
+RUN apt update && apt install -y build-essential pkg-config git ssh curl openssl libssl-dev wget gettext
+
+RUN apt install -y g++-aarch64-linux-gnu libc6-dev-arm64-cross
+RUN apt install -y libgtk-3-dev libwebkit2gtk-4.1-dev libayatana-appindicator3-dev librsvg2-dev librust-alsa-sys-dev
+
+# todo: don't need workdir to be /root/rust
+WORKDIR /root/rust 
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
+RUN . $HOME/.cargo/env && rustup target add aarch64-unknown-linux-gnu 
+RUN . $HOME/.cargo/env && rustup toolchain install stable-aarch64-unknown-linux-gnu
+
+WORKDIR /root/cmake
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.23.1/cmake-3.23.1.tar.gz
```

Comment:
I take it that its easier to download directly vs apt or is this a specific version that you need?

Reply:
In the past I had problems with the version available via `apt` being too low. I did it this way to be safe.